src/main/java/net/minecraftforge/client/event/RenderLivingEvent.java

```diff
@@ -28,35 +28,54 @@
 {
     private final EntityLivingBase entity;
     private final RenderLivingBase<T> renderer;
+    private final float partialRenderTick;
     private final double x;
     private final double y;
     private final double z;
 
+    /*
+        Since adding partialRenderTick causes binary incompatibility,
+        this overload adds a partialRenderTick that simply acts like
+        the game has no values it needs to interpolate.
+     */
+    @Deprecated
     public RenderLivingEvent(EntityLivingBase entity, RenderLivingBase<T> renderer, double x, double y, double z)
     {
+        this(entity, renderer, 1, x, y, z);
+    }
+
+    public RenderLivingEvent(EntityLivingBase entity, RenderLivingBase<T> renderer, float partialRenderTick, double x, double y, double z)
+    {
         this.entity = entity;
         this.renderer = renderer;
+        this.partialRenderTick = partialRenderTick;
         this.x = x;
         this.y = y;
         this.z = z;
     }
 
     public EntityLivingBase getEntity() { return entity; }
     public RenderLivingBase<T> getRenderer() { return renderer; }
+    public float getPartialRenderTick() { return partialRenderTick; }
     public double getX() { return x; }
     public double getY() { return y; }
     public double getZ() { return z; }
 
     @Cancelable
     public static class Pre<T extends EntityLivingBase> extends RenderLivingEvent<T>
     {
-        public Pre(EntityLivingBase entity, RenderLivingBase<T> renderer, double x, double y, double z){ super(entity, renderer, x, y, z); }
+        public Pre(EntityLivingBase entity, RenderLivingBase<T> renderer, float partialRenderTick, double x, double y, double z){ super(entity, renderer, partialRenderTick, x, y, z); }
     }
     public static class Post<T extends EntityLivingBase> extends RenderLivingEvent<T>
     {
-        public Post(EntityLivingBase entity, RenderLivingBase<T> renderer, double x, double y, double z){ super(entity, renderer, x, y, z); }
+        public Post(EntityLivingBase entity, RenderLivingBase<T> renderer, float partialRenderTick, double x, double y, double z){ super(entity, renderer, partialRenderTick, x, y, z); }
     }
 
+    /*
+        Specials uses the deprecated overload because
+        the renderName method that calls it does not
+        provide or use a partialRenderTick value.
```

Comment:
This type of information belongs in the PR explanation on github, not in a comment here.
You should not use the deprecated constructor, it will just have to be changed later. Use the new constructor with a reasonable default value.